src/Benchmarks.ClientJob/project.json

```diff
@@ -1,26 +1,24 @@
 {
-  "version": "1.0.0-*",
+  "version": "1.1.0-*",
   "description": "Class to transfer data to/from the benchmark client.",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/AspNet/Benchmarks"
+  "packOptions": {
```

Comment:
I couldn't get `repository` to work either way ... at the root level or the `packOptions` level. I opened an issue to ask about it: https://github.com/dotnet/cli/issues/3823